docs/articles/apex/writing.md

```diff
@@ -26,51 +37,58 @@ You can preview, view, or go to definitions of:
+When you refresh SObject definitions, the extension uses the default org to generate Apex classes. These classes represent the standard and custom objects that the current user has access to. **Do not** edit these representative Apex classes, which are stored in a hidden directory on your local workstation.
+
+Whenever you refresh SObject definitions, the representative Apex classes are deleted and regenerated. You can modify the SObjects either by updating the objects’ `.object-meta.xml` and `.field-meta.xml` files; Or by making changes declaratively in the default org. After modifying the SObjects, make sure to sync your local project and the default org.
 
-To preview a definition, hold down Ctrl (Windows or Linux) or Cmd (macOS) and hover over the item whose definition you want to see.
+When you launch the Salesforce CLI Integration extension (which is part of the Salesforce Extension Pack) for the first time, `SFDX: Refresh SObject Definitions` command is executed automatically if your project doesn’t contain any Apex classes. After you add or edit standard or custom objects or their fields, make sure to run this command.
 
-To view a definition, right-click the item and select **Peek Definition**, or press Alt+F12.
+## Go To Definitions
+
+Apex extension provides go to definition support for user-defined Apex such as classes and methods, standard objects, and custom objects.
+
+- To preview a definition, press and hold Ctrl (Windows or Linux) or Cmd (macOS) and hover over the item whose definition you want to see.
+- To view a definition, right-click the item and select **Peek Definition**, or press Alt+F12.
+- To jump to the location of a definition, right-click the item and select **Go to Definition**, or press F12.
 
-To jump to the location of a definition, right-click the item and select **Go to Definition**, or press F12.  
 ![Previewing, viewing, and jumping to a definition](../../images/apex_go_to_definition.gif)
 
 ## Find All References
 
-You can find all references to user-defined Apex:
+You can find all references to user-defined Apex such as Classes, Class variables, Enums, Interfaces, Methods, and Properties. To find references, select an item and press Shift+F12; Or right-click the item and select **Find All References**. The reference results are displayed in the left pane of the editor window.
 
-- Classes
-- Class variables
-- Enums
-- Interfaces
-- Methods
-- Properties
+## Outline View
 
-To find references to an item, right-click the item and select **Find All References**, or press Shift+F12.
+The Apex outline view shows the structure of the Apex class or trigger that’s open in the editor.
 
-## Check Syntax Errors in Your Code
+- To view the list of the symbols in the file, press Ctrl+Shift+O (Windows or Linux) or Cmd+Shift+O (macOS).
+- To go to one of the symbols, select it from the list.
+
+![Outline view, showing the symbols in an Apex class](../../images/apex_outline.png)
```

Comment:
We should also point out that there's also an actual outline section in the Explorer View
![Screen Shot 2020-02-06 at 11 09 59 AM](https://user-images.githubusercontent.com/10779873/73970044-546cc280-48d1-11ea-9af8-7cc0ba3efb1b.png)

Reply:
Updated the section with this info.

Reply:
@brpowell, @smaddox-sf , @lcampos , @sfsholden, Addressed all the review comments in Apex Writing and Testing topics. Please do let me know if you have any further feedback for these 2 topics.
Also, I created this branch based on an outdated branch and not the develop branch (facepalm). So I'll delete this pull request and create a new one...